src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -860,4 +860,39 @@
       it { is_expected.to redirect_to(project_show_path(maintenance_project)) }
     end
   end
+
+  describe 'DELETE #clear_failed_comment' do
+    let(:package) { create(:package_with_failed_comment_attribute, name: 'my_package', project: user.home_project) }
+    let(:attribute_type) { AttribType.find_by_name("OBS:ProjectStatusPackageFailComment") }
+
+    before do
+      login(user)
+    end
+
+    context 'with format html' do
+      before do
+        delete :clear_failed_comment, project: user.home_project, package: package
+      end
+
+      it { expect(flash[:notice]).to eq("Cleared comments for packages.") }
+      it { expect(response).to redirect_to(project_status_path(user.home_project)) }
+
+      it 'deletes OBS:ProjectStatusPackageFailComment attribute from package' do
```

Comment:
why do you add a message here and not just

``` Ruby
it { expect(package.attribs.where(attrib_type: attribute_type)).to be_empty }
```

It is shorter and it is just a line so I think it is understandable without the comment. Also, for coherence as the two other tests in this context do not have a message.